src/components/Map.vue

```diff
@@ -136,7 +136,7 @@ export default {
             },
           });
         } else if (accessibleTransports.length === 1) {
-          return resolve(accessibleTransports.slice(0));
```

Comment:
this was a bug, `.slice` returns an array. this would return `["bus"]` for example, which while read correctly by the frontend, was throwing an error in the console.